src/components/OrderPage/index.js

```diff
@@ -1,9 +1,52 @@
 import {useEffect, useState} from "react"
 import ItemCard from "../ItemCard"
 import './OrderPage.scss'
+import OrderCard from "./OrderCard";
+import userEvent from "@testing-library/user-event";
 
 const OrderPage = ({activeRestaurantId}) => {
     const [foodItemData, setItemData] = useState([])
+    const [orderObject, setOrderObject] = useState([])
+
+
+    const addItem = (foodName, price) => {
+        let orderObjectClone = [...orderObject]
+        const orderObjectItem = {foodName : foodName, price : price, qty: 1}
+        let itemOrdered = false
+        orderObject.forEach((orderItem, key) => {
+            if (orderItem.foodName === foodName) {
+                itemOrdered = key
+            }
+        })
+        if (itemOrdered === false) {
+            orderObjectClone.push(orderObjectItem)
+        } else {
+            orderObjectClone[itemOrdered].qty++
+        }
+        setOrderObject(orderObjectClone)
+        console.log(orderObject)
+    }
+
+    const removeItem = (foodName, price) => {
+        let orderObjectClone = [...orderObject]
+        const orderObjectItem = {foodName : foodName, price : price, qty: 0}
+        let itemOrdered = false
+        orderObject.forEach((orderItem, key) => {
+            if (orderItem.foodName === foodName) {
+                itemOrdered = key
+            }
+        })
+        if (itemOrdered === false) {
+            return
+        } else {
```

Comment:
This can be simplified - check `itemOrdered !=== false` and you can ditch the else